product_docs/docs/pgd/5/proxy/index.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: "EDB Postgres Distributed Proxy"
+navTitle: "PGD Proxy"
+---
+
+EDB Postgres Distributed Proxy is a daemon that acts as an abstraction layer between the client application and Postgres. It interfaces with the BDR consensus mechanism to obtain the identity of the current write leader node and redirects traffic to that node.
+
+There is always at least one global group and one data group in the PGD cluster. BDR elects the write leader for each data group which has the `enable_proxy_routing` option set to true. Proxy can be attached to a global group or data group. There could be multiple proxies attached to each group.
+
+PGD Proxy is a TCP layer 4 proxy. It can't encode/decode the PostgreSQL Wire Protocol. Although this may change in future releases.
+
+
+## How it works
+
+Upon starting, PGD Proxy connects to one of the endpoints given in the local config file and fetches DB connection information for all nodes, proxy options like listen address, listen port, routing details like current write leader, etc.
+
+The write leader election and routing is managed by BDR itself. Proxy internally maintains watchers and PubSub mechanism to get leader change notifications on Postgres notify/listen channels.
```

Comment:
not `routing`

Reply:
I don't think we should add this `Proxy internally maintains watchers and PubSub mechanism to get leader change notifications on Postgres notify/listen channels.`

Reply:
concern is only about watcher and PubSub words